scripts/globals/abilities/ancient_circle.lua

```diff
@@ -14,7 +16,14 @@ ability_object.onAbilityCheck = function(player, target, ability)
 end
 
 ability_object.onUseAbility = function(player, target, ability)
-    xi.job_utils.dragoon.useAncientCircle(player, target, ability)
```

Comment:
`xi.job_utils.dragoon.useAncientCircle` already handled the power. there was no need to change it. This is also a regression as it doesn't handle job points. Please revert this file.